spec/lib/admin/system_check/software_version_check_spec.rb

```diff
@@ -27,9 +27,10 @@
 
       context 'when checks are disabled' do
         around do |example|
-          ClimateControl.modify UPDATE_CHECK_URL: '' do
-            example.run
-          end
+          original = Rails.configuration.x.mastodon.software_update_url
```

Comment:
I went with this pattern because it seems most common throughout codebase.

We may want to update these and others to shared a common block-style approach, and/or just stub the values instead of doing the before/after dance.